source/playgrounds.txt

```diff
@@ -177,6 +181,26 @@ of your playground.
 immediately above your selected section. Click this link to test and
 troubleshoot specific lines or sections of your playground.
 
+Code Formatting and Linting Tools
+---------------------------------
+
+MongoDB Playgrounds are proper JavaScript files and can work with popular
+|vscode-short| formatting and linting extensions like
+`Prettier <https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode>`_ 
+and `ESLint <https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint>`_.
+
+If you are using a code formatting extension, MongoDB Playgrounds 
+suggests autocompletion and liniting hints for:
+
+- System variables such as ``$$ROOT`` and ``$$NOW``.
+- ``use`` and ``db`` commands.
+- Collection names, in bracketed notation such as ``db["collection"].find({})``.
+- Alternative commands for shell commands not supported in MongoDB Playgrounds 
+  such as ``use database`` or ``show users``. 
+
+For a full list of autocompletion and linting improvements, see the 
+`v0.11.1 release notes <https://github.com/mongodb-js/vscode/releases>`_.
```

Comment:
This URL points to the most recent version release notes, so when VSCE puts out another release, this link will no longer point to the correct version. Suggest making this URL https://github.com/mongodb-js/vscode/releases/tag/v0.11.1.

Reply:
Done.